### ISSUE TITLE
Triggered build missing env/meta_data blocks

### ISSUE THREAD

**Issue by nolan-fw** [buildkite-plugins/monorepo-diff-buildkite-plugin#32]
I'm struggling trying to pass inputs to my triggered pipeline.

```
steps:
  - label: ":pipeline: Starting pipelines"
    plugins:
      monorepo-diff#v1.0.1:
        diff: ".buildkite/scripts/diff"
        watch:
          - path: "subfolder/"
            config:
              label: ":pipeline: Trigger external"
              trigger: external-build
              build:
                branch: "feat/external-build"
                commit: "HEAD"
                meta_data:
                  current-branch: "${BUILDKITE_BRANCH}"
                env:
                  CURRENT_BRANCH="${BUILDKITE_BRANCH}"
                message: "[${BUILDKITE_BRANCH}] ${BUILDKITE_MESSAGE} Trigger"
```

The output from this running is missing the meta_data and env blocks, and the triggered build is failing on it:
```
INFO[2024-10-15T19:34:23Z] Running diff command: .buildkite/scripts/diff
Generated Pipeline:
steps:
- trigger: external-build
  label: ':pipeline: Trigger external'
  build:
    message: '[feat/triggering-build] Test Trigger'
    branch: feat/external-build
    commit: HEAD
```

Have I misunderstood something in the config block?

**Comment by jihuiyang-x** [comment]
same here, passing env or meta_data does not work.

this is really annoying as it blocks automatic pipeline triggers

**Comment by tnightengale** [comment]
Solved using this: https://github.com/adikari/monorepo-diff-buildkite-plugin/issues/96

TL;DR:
```
env:
    - KEY="VALUE"
```

Gotta love when a company adopts/steals a plugin, and immediately updates the documentation to be wrong. So that users can dig through docs on the the original project. 🙃

**Comment by pzeballos** [comment]
Hey folks! Sorry for the delay here.
Thanks for flagging this; we totally get how frustrating it can be when documentation isn’t as clear as it should be. There was an example under the `env` section in the README showing the correct syntax (list of strings), but it clearly wasn’t easy to spot, so that’s on us.
We’ve made a few recent changes to improve readability and will keep working on making the docs easier to follow.
